addons/point_of_sale/static/tests/tours/helpers/ProductScreenTourMethods.js

```diff
@@ -391,21 +391,25 @@ export function addCustomerNote(note) {
 }
 
 export function addInternalNote(note) {
-    return inLeftSide(
-        [
-            {
-                content: "click more button",
-                trigger: ".mobile-more-button",
-                mobile: true,
-            },
-            {
-                content: "click customer note button",
-                trigger: '.control-buttons .control-button span:contains("Internal Note")',
-            },
-            TextAreaPopup.inputText(note),
```

Comment:
`...( note ?  TextAreaPopup.inputText(note) : [] ),
`

Reply:
Hello @vlst-odoo :wave: 
Thanks for the review.
Requested changes are done.